integrationtests/tests/fixtures/real.rs

```diff
@@ -18,36 +19,69 @@ use serde::Serialize;
 use crate::fixtures::{BitcoinTest, LightningTest};
 
 pub struct RealLightningTest {
-    rpc_gateway: LightningRPC,
-    rpc_other: LightningRPC,
+    rpc_gateway: ClnRpc,
+    rpc_other: ClnRpc,
```

Comment:
Maybe its a good idea to put those into a `RefCell` (or `Mutex` if needed) so we can avoid all the `&mut self` later. Not sure if this makes sense though, what do you think @dpc @jkitman ?

Reply:
Yes that is a pattern used elsewhere, we want to keep the `tests.rs` code as simple as possible, so using `RefCell` or `block_on` is fine to achieve that because we are just dealing with integration tests where we control how the fixtures are used.